scripts/supported-releases.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Prefers GNU semantics for sed, grep, sort, etc
+
+supported_versions() {
+    local one_year_ago
+    one_year_ago=$(date +%s -d "1 year ago" 2> /dev/null || date -j -v-1y +%s)
+    # Filter out version X.Y.0 happening before a year ago
+    while IFS='' read -r line; do
+      local release
+      local released_date
+      released_date=$(echo "${line}" | awk '{print $2}')
+      if (( released_date > one_year_ago)); then
+        release=$(echo "${line}" | awk '{print $1}' |awk -F/ '{print $3}' | sed 's/\.0$//')
```

Comment:
Does `sed` works in this case as it should for both Mac and Linux?

Reply:
The script worked initially for me in Mac, then failed in the ubuntu CI. I used a local ubuntu container to debug it, but the only thing affected was the grep regex expression, as Mac uses POSIX regex and GNU grep uses Perl' s regex.
The current expression seem to work properly in both.
I did not found any issues with sed in Mac or the CI.